src/scripts/navbar.js

```diff
@@ -0,0 +1,61 @@
+window.onscroll = function () {
+  scrollFunction();
+};
+
+let navbar = document.getElementById("navbar");
+const hamburgerMenu = document.getElementById("menuIcon");
+const overlay = document.querySelector(".overlay");
+const navbarRight = document.getElementById("navbar-right");
+const dropdown = document.querySelectorAll(".dropdown");
+
+function scrollFunction() {
```

Comment:
You should avoid using `magic numbers` in your code for better readability and debugging. 
```
const bodyScrollTriggerPoint = 5;

if (document.body.scrollTop > bodyScrollTriggerPoint) {...}
``` 

You could put long if statements in variables for better readability:

```
const isScrolled = document.body.scrollTop > 5 || document.documentElement.scrollTop > 5;

if (isScrolled) {...}
```